README.md

```diff
@@ -49,10 +49,13 @@ Run:
 
 ```bash
 docker run -it -p 8000:80 asyncapi/studio
+docker run -it -p 8000:80 -v "$(pwd)/another/path/to/asyncapi:/usr/share/nginx/html/docs/asyncapi" asyncapi/studio
```

Comment:
Could we split that line to separate paragraph to describe how to add volume with AsyncAPI spec, something like:

```md
To mount a specification, use the command:

\```bash
docker run -it -p 8000:80 -v "$(pwd)/another/path/to/asyncapi:/usr/share/nginx/html/docs/asyncapi" asyncapi/studio
\```

> **NOTE**: The file should be named `asyncapi.yaml` or `asyncapi.json` in the specified location.
```

Please remove `\` before backticks 😅

Reply:
idk who or what added that back slash 🤣 will immediately cover the shame